DOCS-3930 Moving Around the Synthetic Testing Tunnel (#15316)
DataDog/documentation: content/en/continuous_testing/cicd_integrations/configuration.md

---

> title: Synthetics and CI/CD Configuration

Reviewer: ```suggestion
title: Continuous Testing and CI/CD Configuration
```

kind: documentation

> description: Configure Synthetics to run tests in your CI/CD pipelines.

Reviewer: ```suggestion
description: Configure Continuous Testing to run tests in your CI/CD pipelines.
```

aliases:
  - /synthetics/cicd_integrations/configuration
further_reading:
- link: "https://www.datadoghq.com/blog/datadog-github-action-synthetics-ci-visibility/"
  tag: "Blog"

>   text: "Use Datadog's GitHub Action to add Synthetic testing to workflows"

Reviewer: ```suggestion
  text: "Use Datadog's GitHub Action to add continuous testing to workflows"
```
